### DISCUSSION TITLE
能否使用命令行来控制模块作用域？

### DISCUSSION THREAD

**Discussion by AshenOneYe** [LSPosed/LSPosed#2640] [Q&A] [answered]
用于自动化测试，每次测试时会将目标app卸载重装，导致之前勾选的模块作用域丢失，每次需要重新勾选，希望能够记住勾选，或者支持使用adb命令的方式来手动勾选

**Comment by vvb2060** [comment, marked as answer]
不会做，这会导致绕过用户许可

**Comment by alinnse** [comment]
https://github.com/mywalkb/LSPosed_mod

这个可以

**Reply by AshenOneYe** [reply]
感谢
